src/modules/wallet/components/AllTokens/AllTokens.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable max-statements */
 import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { useLocation } from 'react-router';
```

Comment:
I think we should only import from  `react-router-dom`, and when migration is complete remove `react-router` completely from our package.json. `react-router-dom` is a super-set of `react-router` no need for both